src/gui/SearchWidget.cpp

```diff
@@ -41,9 +41,11 @@ SearchWidget::SearchWidget(QWidget* parent)
     connect(m_searchTimer, SIGNAL(timeout()), this, SLOT(startSearch()));
     connect(this, SIGNAL(escapePressed()), m_ui->searchEdit, SLOT(clear()));
 
-    new QShortcut(Qt::CTRL + Qt::Key_F, this, SLOT(searchFocus()), nullptr, Qt::ApplicationShortcut);
+    new QShortcut(QKeySequence::Find, this, SLOT(searchFocus()), nullptr, Qt::ApplicationShortcut);
     new QShortcut(Qt::Key_Escape, m_ui->searchEdit, SLOT(clear()), nullptr, Qt::ApplicationShortcut);
 
+    m_ui->searchEdit->setPlaceholderText(tr("Search (%1)...", "Search placeholder text, %1 is the keyboard shortcut")
+                                            .arg(QKeySequence(QKeySequence::Find).toString(), QKeySequence::NativeText));
```

Comment:
@droidmonkey Still returning CTRL 

`QKeySequence::NativeText` should be an argument of `.toString()` instead of `tr().arg()`.

Reply:
Fixed, I made a stupid undo right before commit.

Reply:
Man I am striking out today! Good catch 😢